package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-calendar-timeline",
-  "version": "0.17.0",
+  "version": "0.17.1",
```

Comment:
For future PRs, don't worry about updating version here. I'll update that as a part of a release

Reply:
Oh, I did it, really? Sorry, just a habit)